docs/README.md

```
@@ -0,0 +1,174 @@
+```terminal
+$ pnpm install
+```
+
+# Running the Development Server
+
+To preview your changes as you edit the files, you can run a local development server that will serve your website and it will reflect the latest changes.
+
+The command to run the server is:
+
+```console
+$ pnpm start
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)